xcm/src/v0/junction.rs

```diff
@@ -94,11 +94,7 @@ pub enum Junction {
 	/// An indexed parachain belonging to and operated by the context.
 	///
 	/// Generally used when the context is a Polkadot Relay-chain.
-	///
-	/// There is also `Parachain` which can be used in tests to avoid the faffy `{ id: ... }` syntax. Production
-	/// code should use this.
```

Comment:
I can't understand what this sentence means, I don't see `Parachain` types to be used in tests.
But I expect that it is not needed now that we fixed the syntax.

Reply:
Yeah - that's a stale comment.